frontends/web/src/routes/account/summary/accountssummary.tsx

```diff
@@ -244,7 +244,11 @@ class AccountsSummary extends Component<Props, State> {
                                         { accounts.length > 0 ? (
                                             accounts.map(account => this.balanceRow(account))
                                         ) : (
-                                            <p>{t('accountSummary.noAccount')}</p>
+                                            <tr>
+                                                <td colSpan={3} className={style.loadingAccounts}>
+                                                    {t('accountSummary.loadingAccounts')}
```

Comment:
How about reusing a more generic [loading](https://github.com/digitalbitbox/bitbox-wallet-app/blob/c6aab96/frontends/web/src/locales/en/app.json#L893) key? I don't think it matters that much here, what's loading.